mozjs/js/rust/src/jsslot.js

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+function String_numslot() {
+    return 1;
+}
+
+function Vec_numslot() {
+    return 3;
+}
```

Comment:
Yes. I need to copy those and all the generated js file in the Utilities.js file to make this patch work. a temporary solution. If there's way to specify what to load in the buildin directory, files can be just move to those dir instead.